modules/stitching/src/stitcher.cpp

```diff
@@ -541,6 +540,102 @@ Stitcher::Status Stitcher::estimateCameraParams()
     return OK;
 }
 
+Stitcher::Status Stitcher::setTransform(InputArrayOfArrays images, const std::vector<detail::CameraParams> &cameras) {
```

Comment:
Probably the following code is excessive, as the same result could be achieved by reusing an initialized (through estimateTransform) stitcher's composePanorama with other images (i.e. different from those passed to estimateTransform). Providing the setter to the homographies (cameras array) would be sufficient.